addons/website/static/tests/tours/snippet_background_edition.js

```diff
@@ -182,6 +182,11 @@ wTourUtils.clickOnSnippet(snippets[0]),
 }),
 
 // Now, add an image on top of that color combination + gradient
+{
+    // Close the palette before selecting a media.
+    trigger: '.snippet-option-ColoredLevelBackground we-title',
+    content: 'Close palette',
+},
```

Comment:
Mmh, in fact I see this is a test tour (not an users tour), why does this require a new step if it is not currently breaking? Commit message does not seem to explain that?

Reply:
The palette must now be closed before the next `changeOption` step can be activated.
I have updated the message accordingly.

Reply:
Mmh ok, forgot that the next step was modified indirectly. The comment in commit was not entirely necessary then, just me that was dumb :wink: